### PR TITLE
[v3] Update next for deprecation replacement

### PR DESCRIPTION
**Pull request by jace-roell**
**What It Does**
Modified showMsgWhenDeprecated() to allow an empty string as a parameter when no replacement is available for deprecated command to print an alternative message.

**Review Checklist**
I certify that I have:
- [x] tested my changes
- [x] added/updated automated tests
- [x] updated the changelog
- [x] followed the [contribution guidelines](https://github.com/zowe/zowe-cli/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: packages/imperative/src/utilities/src/CliUtils.ts [hunk 1]
```diff
@@ -378,20 +392,14 @@ export class CliUtils {
...
-            }
-
+            const oldCmd = handlerParms.positionals.join(" ");
             // display the message
             handlerParms.response.console.error("\nWarning: The command '" + oldCmd + "' is deprecated.");
-            handlerParms.response.console.error("Recommended replacement: " +
-                handlerParms.definition.deprecatedReplacement);
+            // Use consolidated deprecated message logic
+            const deprecatedMessage = CliUtils.generateDeprecatedMessage(handlerParms.definition);
+            handlerParms.response.console.error(deprecatedMessage);
```
Comment (t1m0thyj) [inline packages/imperative/src/utilities/src/CliUtils.ts:402]: Previously the `deprecatedReplacement` was printed as-is, now with the consolidated logic it will have newlines removed: `cmdDefinition.deprecatedReplacement.replace(/\n/g, " ")`. Is this change intentional?

Comment (t1m0thyj) [reply packages/imperative/src/utilities/src/CliUtils.ts:402]: Previously `DefaultHelpGenerator` stripped out new lines when parsing the deprecation message, and `CommandProcessor` kept them. I like that the new behavior is consistent 👍 

https://github.com/zowe/zowe-cli/blob/e10af2e2eaeafcbe3f68762405746a8b801bc7b0/packages/imperative/src/utilities/src/CliUtils.ts#L376

Not sure which we prefer - stripping out new lines vs keeping them. It may not matter much since I don't recall seeing a deprecation message that contains new lines 😋

**Review by t1m0thyj — APPROVED** [review]
LGTM, thanks @jace-roell!

**Review by gejohnston — APPROVED** [review]
